### ISSUE TITLE
Estou com medo de entrar na área 

### ISSUE THREAD

**Issue by silvaMatheus** [frontendbr/forum#833]
Recentemente terminei um site e quando entreguei o cliente falou "descobri uma ferramenta nova pra criar sites se eu tivesse conhecido antes nem teria contratado vc" bom ele pagou o site eu entreguei ok... Mas até agora estou pensando como vai continuar a área de front daqui um tempo? Estou  "entrando" na área agora estou estudando MUITO mas agora estou com um certo medo de investir tempo e dinheiro em algo q daqui uns qualquer pessoa vai poder fazer e não vai precisar mais das minhas habilidades.  Poderiam tirar esse meu medo ? Kkkk tô gostando muito de estudar essa área

Obs: me mandaram esse vídeo agora a pouco falando sobre uma ferramenta do [Google]( https://m.youtube.com/watch?v=vkOxSXquUb8   )

**Comment by ewerto0n** [comment]
Fiz uma rápida leitura e vi o vídeo. Brevemente falando, isso vem para pequenas empresas, quem está começando. É algo que progride igual a qualquer tecnologia e os profissionais tem que progredir junto. Então, pensando assim, temos a entrada de mais empresas online certo? Vamos pensar nisso como motivação?! Opinem!

**Comment by marcusvcribeiro** [comment]
Se fosse por isso, não teríamos uma variedade de empregos que existem hoje
em dia que a tecnologia poderia simplesmente substituir.
Estude, foque no que tu quer, cresça como profissional e esqueça isso.
Sempre haverá boatos, O Frontend vai acabar com o backend, backend vai
servir somente para consulta em bancos, ferramentas que geram html/css/js
irá acabar com o frontend, e blablabla.
O mercado ta ai, pagando bem(querendo ou não, dev ganha muito bem em
comparação a maioria das outras profissões.), então esquece esse medo, que
não vai te levar a lugar algum.

Em 30 de novembro de 2017 13:56, Ewerton Nepomuceno (nepu) <
notifications@github.com> escreveu:

**Comment by kvnol** [comment]
> @marcusvcribeiro: então esquece esse medo, que não vai te levar a lugar algum

Exato! E, se um dia acabar, paciência, vamos fazer outra coisa ahahahah

**Comment by PedroFelipe** [comment]
O frontend é muito mais que landing pages ou sites institucionais. Continue estudando, a sua especialização vai te trazer no futuro projetos além do que pode ser criado com o Wix ou uma ferramenta gratuita do Google.

**Comment by bernardodiasc** [comment]
Fazer café é muito fácil, vc esquenta água e passa ela pelo pó de café, pronto.

Mesmo assim café é vendido aos montes todos os dias.

Não se engane, até mesmo doutores do escambau a quatro podem ter coisas mais importantes pra fazer do que um website, nessa hora vão contratar alguém pra executar a tarefa.

E se alguém disser:

> descobri uma ferramenta nova pra criar sites se eu tivesse conhecido antes nem teria contratado vc

Voce pode responder:

> vai lá, boa sorte, mas anota meu email pra quando voce se der conta da má escolha que fez e precisar do meu serviço novamente.

Agora, falando uma opinião muito pessoal sobre isso, esse tipo de cliente é o tipo que eu me recuso a trabalhar, são pessoas que não tem ideia o trabalho que dá fazer qualquer coisa em informatica e tambem nao dao valor para o esforço alheio, o famoso "fácil é o trabalho dos outros". E uma dica de mestre (sim, mestre, pois me custou muito para aprender isso), nunca recuse um trabalho, quando não quiser pegar o serviço (pelo motivo de cliente ruim), cobre um preço absurdo e não abaixe. Isso é muito importante, o cliente ruim não vai fechar negócio com voce mas voce vai ter imposto seu valor.

**Comment by tcelestino** [comment]
Estou na área há muito tempo e posso te dizer que esse papo do "fim" de qualquer profissão que envolve tecnologia sempre será um "papo". 

Sempre existiu geradores de sites (hpg, kit.net, etc...) e sempre houve essa discussão de que na época, o mercado de "web design" e "webmaster" poderiam acabar. Hoje estamos ai!

Infelizmente, para algumas coisas, realmente não é necessário um profissional (já citaram ai em cima o porque), mas existem outras demandas, outros porém que ainda será necessário um ser humano para realizar o trabalho braçal. E se você se focar em trabalhar em serviços e não só apenas em criar hotsites, landing page, customização de fanpages (ainda existe isso? hahahah), etc..., vai ter um pensamento diferente.

No final, não tenha medo! O máximo que você pode acontecer é você aprender e seguir na luta. 👍

**Comment by LFeh** [comment]
Toda minha opinião está nesse artigo que escrevi em algum momento desse ano.

## [O inicio, o fim e o meio do Desenvolvimento Front-End](https://medium.com/@lfeh/o-inicio-o-fim-e-o-meio-do-desenvolvimento-front-end-dfc5a123b90f)

Mano... as profissões mudam e os profissionais (as boas e os bons) se readaptam. Não se intimide. 

Temos sim que torcer pra que a tecnologia evolua cada vez mais e que nossa profissão se torne obsoleta. Assim que o mundo evolui ;)

**Comment by silvaMatheus** [comment]
Vlw pessoal muito obg precisava  de um incetivo pra continuar ... @LFeh  ótimo artigo Man

**Comment by vitocaio** [comment]
Sempre vão precisar de um Front para desenvolver o Gerador de sites k

**Comment by alinebastos** [comment]
Eu já tive uma quase cliente pra um freela, conversamos bastante sobre o que ela queria, passei o orçamento e ela ficou de pensar. Uns dias depois ela me procurou pedindo desculpas e dizendo que ia fazer com o Wix, pq ela tava com pouca grana. Tranquilo, falei que qualquer coisa que ela precisasse que me chamasse e era isso. Jamais pensei que eu iria ficar sem trabalho por causa do Wix, são públicos-alvo diferentes.

A profissão de frontend já mudou bastante desde que eu comecei, agora está cada vez mais complexa e completa, não é só mais codificar o layout. Então essas ferramentas que aparecem não vão acabar com a profissão, pois elas são feitas pra quem precisa de algo rápido, fácil e barato.

**Comment by doug2k1** [comment]
Você pode comprar remédio na farmácia, sem receita, que resolve alguns casos. Mas isso não acaba com a profissão de médico.

**Comment by felipebernardes** [comment]
fala @silvaMatheus ! blz?

a área nunca vai morrer. mas SEMPRE vai continuar mudando. basta você acompanhar e se atualizar (participar de foruns, ter projetos pessoais paralelos, se importar em entender conceitos e não só saber como usar frameworks, etc.)
